modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/classvars.mustache

```diff
@@ -112,6 +112,12 @@
             openapi_types (dict): The key is attribute name
                 and the value is attribute type.
         """
+{{#oneOf}}
+{{#-first}}
+        return {}
+{{/-first}}
+{{/oneOf}}
+{{^oneOf}}
```

Comment:
This should be removed after we find a solution for https://github.com/OpenAPITools/openapi-generator/pull/6674/files#r440966593.
```suggestion
```

Reply:
So this omits variables that are defined in:
```
components:
  schemas:
    ComposedModelWithProperties:
      properties:
        - color:
        type: string
      oneOf:
        - Cat
        - Dog
```
Can you add an example like I have shown above and show that color is present in ComposedModelWithProperties.openapi_types?